<launch>
  <node pkg="mechanism_control" type="spawner.py"
        args="$(find caster_life_test)/controllers.xml" />
  <node pkg="caster_life_test" type="periodic_drive.py" />
</launch>